tests/Unit/Controllers/WorkItemControllerTest.php

```
@@ -2373,6 +2373,57 @@ public function generateJson(string $prompt, string $input, array $options = [])
+            $this->auth,
+            $this->db,
+            $this->config,
+            $this->storyAiServices(),
+            $this->workItemImprovementWorkflow(),
+            new RoadmapCoverageWorkflow()
+        );
+        $ctrl->generateForOkr();
+
+        $this->assertSame('/app/diagram?project_id=5', $this->response->redirectedTo);
+        $this->assertStringContainsString('No strategy diagram found', $_SESSION['flash_error'] ?? '');
+    }
```

PHPMD (2.15.0)

[error] generateForOkrRedirectsToDiagramWhenProjectHasNoDiagram accesses the super-global variable $_SESSION. (undefined)

(Superglobals)